Copyright Pass/Pruning | Part 1 - Clothing

SweptWasTaken opened the pull request:
Phew

I'm splitting this PR into multiple parts so it's less strain on reviewers. This one is dedicated exclusively to the clothing entities and .rsis.

The goal of this PR was to update all meta.jsons with correct licensing/copyright information, but I ended up just doing more general pruning. 

- Standardized meta.json formats across all clothing .rsis
- Pruning of content that was either duplicated/cev-eris stuff that isn't relevant/ugly looking stuff
- ID Standardization
- Made the clothing folder structure a little more sensible
- Used best-practice YAML formatting.
- More QoL stuff

I have removed a lot of stuff in the process, so i'll present a list here.
![explorer_2020-11-12_18-19-52](https://user-images.githubusercontent.com/49448379/99020561-c7892080-2556-11eb-88c1-0f66fcb02b3f.png)

![98yh98y](https://user-images.githubusercontent.com/49448379/99020750-2d75a800-2557-11eb-990f-fdab1b8e40e3.jpg)

metalgearsloth commented on Resources/Prototypes/Entities/Clothing/OuterClothing/base.yml:
```diff
@@ -0,0 +1,20 @@
+- type: entity
+  parent: Clothing
+  id: ClothingOuterBase
+  abstract: true
+  components:
+  - type: Clothing
+    Slots:
+    - outerclothing
+  - type: Sprite
+    state: icon
+
+- type: entity
+  parent: OuterclothingBase
```
Should be ClothingOuterBase?

metalgearsloth reviewed (changes requested):
I've seen a couple typos under outer clothing but I'll suss the rest later.

SweptWasTaken commented:
Ready for review.

For reviewers, no I haven't hidden anything in this pr. I'm not a monster.

The **majority** of the files changed are me shifting around the organization of the rsis in the Clothing folder. To review, just take a quick look at the format i've placed them in and lemme know if you dislike it.

When editing the meta.jsons of every rsi, I meticulously updated their copyright, standardized their layout, and made some optimizations to them. Take a look at how i've done one rsi and the rest will look like that.

I've made changes to saltern.yml and a bunch of Fills only to fix the ID conflicts caused by this PR.

Gone ahead with my ID verbosity thing so that most IDs point to the file structure the entities are in. This is good for devs, the entity spawner and helps locate entities better.

Mirrored the rsi and entity folder structures.

**The actual additions i've made**
-Updated some sprite icons newer versions from tgstation.
-Added a bunch more glasses
-Added a couple things to the Roles like cosmetics to spawn with (And also fixed ID conflicts)

There's probably some more small stuff in there, but that's p much it.

metalgearsloth commented on Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:
```diff
@@ -1,206 +1,180 @@
-  description: Heavily armored black and green suit for long combat situations, also provides protection from low-pressure/vacuum damage.
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitSyndie
+  name: blood red hardsuit
```
```suggestion
  name: blood-red hardsuit
```

metalgearsloth commented on Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:
```diff
@@ -0,0 +1,550 @@
+
+- type: entity
+  parent: ClothingUniformJumpsuitChiefEngineer
+  id: S
+  name: s
+  components:
+  - type: Sprite
+    state: s
+  - type: Clothing
+    HeldPrefix: s
```
S moment

metalgearsloth commented on Resources/Prototypes/Entities/Clothing/Uniforms/scrubs.yml:
```diff
@@ -0,0 +1,32 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: UniformScrubsPurple
+  name: purple scrubs
```
Should these have colour on them? Just because stuff like the bio suit for example doesn't it.

metalgearsloth commented on Resources/Prototypes/Entities/Clothing/Uniforms/base.yml:
```diff
@@ -0,0 +1,20 @@
+  - type: Sprite
+    state: icon
+  - type: Clothing
+    Slots: [innerclothing]
```
Should be new line IMO.

SweptWasTaken replied:
I'm assuming you mean 

`Slots:
 - innerclothing`

metalgearsloth commented on Resources/Prototypes/Roles/Jobs/Command/captain.yml:
```diff
@@ -11,6 +11,7 @@
   # All of em.
   # Could probably do with some kind of wildcard or whatever to automate this.
   # HELL FUCKING YEAH WE COULD
+  # Guys please don't fight
```
;-;

metalgearsloth commented on Resources/Textures/Clothing/Uniforms/Jumpsuit/rainbow.rsi/meta.json:
```diff
@@ -0,0 +1,27 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```
yay rainbow jumpsuit's back

metalgearsloth reviewed (approved):
After build passes I guess it's good to merge... ngl I just random sampled the sprites.

ike709 commented on Resources/Maps/saltern.yml:
```diff
@@ -1,7 +1,7 @@
-  name: DemoStation
-  author: Space-Wizards
+  name: Saltern
+  author: Buncha sickass mfers
```
🤔

DrSmugleaf commented on Resources/Prototypes/Catalog/Fills/belt.yml:
```diff
@@ -0,0 +1,53 @@
+    contents:
+      - name: Soap #Make a soap group and pick between when i'm not lazy
+      - name: SprayBottleSpaceCleaner
+      - name: # GrenadeChem
```
Yes, fixed

DrSmugleaf reviewed (changes requested):
Address the current reviews and fix the error when the game starts up and it's good to merge.

SweptWasTaken commented:
You monster you addressed the reviews yourself

DrSmugleaf commented:
> You monster you addressed the reviews yourself

I had to do it to em

SweptWasTaken commented:
Addressed all reviews.

DrSmugleaf commented:
Some of the clothing removed with this PR:

- HatBandana, HatBandcamo, HatBearpelt, HatCowboy (hats.yml vending machine)
- UniformAssistant (suits.yml vending machine)
- OuterclothingAcolyte, OuterclothingBedsheet, OuterclothingChurchcoat (chapel.yml vending machine)

metalgearsloth commented:
yeet

SweptWasTaken commented:
damn shawty ok
